src/main/java/pl/edu/agh/firecell/model/Material.java

```diff
@@ -7,25 +7,28 @@
 public enum Material {
     // TODO:
     // consider putting them into constant
-    AIR(0, 0, 0, FLUID, 900),
-    WOOD(3, 250, 0, SOLID, 550);
+    AIR(0, 0, 0, FLUID, 900, 0),
+    WOOD(3, 250, 0, SOLID, 550, 30);
 
     private final double ignitionTemperature;
     private final double autoIgnitionTemperature;
     private final double fireSpreadSpeed;
```

Comment:
This field is not used. It should be deleted, but if you're going to use it in future you can leave it.